src/basic-syntax/doc-comments.md

```diff
@@ -0,0 +1,27 @@
+# Documentation Comments
+
+All language items in Rust can be documented using special `///` syntax.
+
+```rust,editable
+/// Determine whether the first argument is divisible by the second argument.
+///
+/// If the second argument is zero, the result is false.
+fn is_divisible_by(lhs: u32, rhs: u32) -> bool {
+    if rhs == 0 {
+        return false;  // Corner case, early return
+    }
+    lhs % rhs == 0     // The last expression in a block is the return value
+}
+```
+
+The contents are treated as Markdown. All published Rust code is automatically
```

Comment:
More specifically, all Rust library crates (not binary crates) published to crates.io.